User Guides/19-Restarting a Control Server Deployment.md

```diff
@@ -0,0 +1,22 @@
+# Restart a Failed Control Server
+The Control Server as the brain box of the entire system is still susceptible to failure. The following steps describes how to re-deploy a failed Control Server.
+
+#### Restart Control Server Deployment
+1. Take a note of your resource group name. 
+
+2. Login to [Azure](portal.azure.com)

```

Comment:
Broken link. Add protocol

Reply:
Also, to be inline with the other docs please call it "Azure portal".

Reply:
Done